subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/InteractionContext.kt

```diff
@@ -30,11 +30,13 @@ class SimpleInteractionContext(
     private val precondition: () -> Unit = {}
 ) : InteractionContext {
 
+    private var inPrecondition = false
```

Comment:
@dsvoronin I supposed that all checks happen on UI thread.
Or do we need concurrency safety?